apis/storage/volumeclass_types.go

```diff
@@ -21,6 +21,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// Max throughput in bytes per seconds.
+	ResourceBPS corev1.ResourceName = "bps"
+	// Max IOPS in input/output operations per second.
+	ResourceIOPS corev1.ResourceName = "iops"
+	// Dynamic resource limits flag: limits  per GB of volume.
+	ResourceDynamic corev1.ResourceName = "dynamic"
```

Comment:
we should call this `maxSize`

Reply:
With `ResourceDynamic` I wanted to express that the limits of the `VolumeClass` are absolute limits or 'dynamic' limits. 
In case if the `ResourceDynamic` key is present the limits are multiplied with the requested size of the `Volume`. 

It might be better to put such a flag  in the spec of the `VolumeClass` itself.